src/main/steps/common/documents/upload/upload-your-documents/template.njk

```diff
@@ -63,7 +63,7 @@
       <div class=" govuk-!-margin-bottom-3">
         {% if errorMessage !== null %}
           <div class="govuk-form-group--error">
-          {% endif %}
+        {% endif %}
```

Comment:
why there are changes in src/main/steps/common/documents/upload/upload-your-documents/template.njk file?

Reply:
Corrected the spacings nothing else here